grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -583,6 +587,40 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
     );
   }
 
+  renderLabels(item: AlertType) {
+    if (!item.labels.length) {
+      return null;
+    }
+
+    return (
+      <TooltipBadge
+        tooltipTitle=""
```

Comment:
I can see that we're passing empty string in multiple places to this component:
![image](https://github.com/grafana/oncall/assets/12073649/f963124e-ca15-42fe-b157-e33ca639caf9)

How about we make this prop optional in `TooltipBadge` and then we get rid of passing empty string in those cases?